egs/csj/s5/local/nnet3/run_tdnn_baseline.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+
+# This version of the TDNN system is being built to have a similar configuration
+# to the one in local/online/run_nnet2.sh, for better comparability.
+
+. cmd.sh
+
+
+# At this script level we don't support not running on GPU, as it would be painfully slow.
+# If you want to run without GPU you'd have to call train_tdnn.sh with --gpu false,
+# --num-threads 16 and --minibatch-size 128.
+
+if [ -e data/train_dev ] ;then
+    dev_set=train_dev
+fi
+
+stage=0
+train_stage=-10
+dir=exp/nnet3/nnet_tdnn_c
+. cmd.sh
+. ./path.sh
+. ./utils/parse_options.sh
+
+
+if ! cuda-compiled; then
+  cat <<EOF && exit 1 
+This script is intended to be used with GPUs but you have not compiled Kaldi with CUDA 
+If you want to use GPUs (and have them), go to src/, and configure and make on a machine
+where "nvcc" is installed.
+EOF
+fi
+
+local/nnet3/run_ivector_common.sh --stage $stage || exit 1;
+
+if [ $stage -le 8 ]; then
+  if [[ $(hostname -f) == *.clsp.jhu.edu ]] && [ ! -d $dir/egs/storage ]; then
+    utils/create_split_dir.pl \
+     /export/b0{3,4,5,6}/$USER/kaldi-data/egs/csj-$(date +'%m_%d_%H_%M')/s5/$dir/egs/storage $dir/egs/storage
+  fi
+
+
+  steps/nnet3/train_tdnn.sh --stage $train_stage \
+    --num-epochs 8 --num-jobs-initial 2 --num-jobs-final 10 \
+    --splice-indexes "-1,0,1  -2,1  -4,2 0" \
```

Comment:
--splice-indexes format is wrong (there is a semicolon) and --io-opts is no longer supported.